apps/web/src/app/[orgShortcode]/settings/page.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import { useGlobalStore } from '@/src/providers/global-store-provider';
+import SettingsSidebarContent from './_components/settings-sidebar';
+import { useIsMobile } from '@/src/hooks/use-is-mobile';
+import { redirect } from 'next/navigation';
+
+export default function Page() {
+  const orgShortcode = useGlobalStore((state) => state.currentOrg.shortcode);
+  const isMobile = useIsMobile();
+  if (!isMobile) {
+    redirect(`/${orgShortcode}/settings/user/profile`);
```

Comment:
you can't use redirect here, you need to use `router.push`

Reply:
redirect works
whats the downside to using it?

Reply:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/c9c76c52-2ab6-4658-871e-9f94a67f7fa7">

client side code is recomended to use router

Reply:
I was a bit wrong here, its being called during render, using router.push would cause issues here not redirect